packages/chart/src/components/LineChart.jsx

```diff
@@ -18,6 +18,8 @@ export default function LineChart({ xScale, yScale, getXAxisData, getYAxisData,
   const cleanedData = cleanData(data, config.xAxis.dataKey)
   const { yScaleRight } = useRightAxis({ config, yMax, data, updateConfig })
 
+  //console.log("LineChart ###CleanedData=", cleanedData)
```

Comment:
nit: remove comments unless they should stay in.

Reply:
Removed it.  No just accidentally left in.

Reply:
Removing now, looks like it snuck back in.